code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -1898,11 +1898,20 @@ datum
 			mix_phrase = "Large white crystals precipitate out of the mixture."
 			mix_sound = 'sound/misc/drinkfizz.ogg'
 
+		iron_oxide
+			name = "Iron Oxide"
+			id = "iron_oxide"
+			result = "iron_oxide"
+			required_reagents = list("iron" = 1, "oxygen" = 1, "water" = 1)
+			result_amount = 3
+			mix_phrase = "The iron rapidly rusts."
+			required_temperature = T0C + 75
```

Comment:
To make it slightly more cumbersome to produce I suggest changing `required_temperature` to be _strictly_ higher than T0C+100 instead (this will require switching out water as a required reagent). Also should consider lowering `result_amount`.